platform/linux-generic/odp_traffic_mngr.c

```diff
@@ -2607,6 +2607,10 @@ static int tm_capabilities(odp_tm_capabilities_t capabilities[],
 		per_level_cap->max_priority       = ODP_TM_MAX_PRIORITIES - 1;
 		per_level_cap->min_weight         = ODP_TM_MIN_SCHED_WEIGHT;
 		per_level_cap->max_weight         = ODP_TM_MAX_SCHED_WEIGHT;
+		per_level_cap->min_burst          = 1U;
+		per_level_cap->max_burst          = 0xFFFFFFFFU;
```

Comment:
I think UINT32_MAX would be better here than the literal which has more F's than I can easily count (which is about 4).